lib/github-pages-health-check.rb

```diff
@@ -9,8 +9,6 @@
 require "resolv"
 require "timeout"
 require "octokit"
-require "json"
-require "yaml"
```

Comment:
If using the YAML approach for `to_s`, you'll want to re-require `yaml` (or maybe lazy require it in the method?)